practice_app/api/models.py

```diff
@@ -1,3 +1,57 @@
 from django.db import models
+from django.contrib.auth.models import User
+
+import json
 
 # Create your models here.
+
+class Paper(models.Model):
+    paper_id = models.AutoField(primary_key=True)
```

Comment:
As I remember, the source and the third_party_id were chosen as composit key in the meetings.
paper_id is not a problem, however, the combination of the third_party_id and source must be unique nevertheless.

Reply:
We decided to have a single paper_id in the meeting because it causes multiple foreign keys in all other tables.

Reply:
Yeah but I can still add a unique constraint for the third_party_id and source keeping the paper_id field. Other tables can access with paper_id.

Reply:
I added a unique constraint for third_party_id and source keys.